<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportCitcomsResolvedTopologyOptionsWidget</class>
 <widget class="QWidget" name="ExportCitcomsResolvedTopologyOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_shapefile_dateline_wrap" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_plate_polygons">
     <property name="title">
      <string>Plates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_export_individual_plate_polygon_files">
        <property name="text">
         <string>Export each Plate polygon to a separate file (%P='plate_'+rotation id)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_plate_polygons_to_single_file">
        <property name="text">
         <string>Export all Plate polygons to a single file (%P='platepolygons')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_plate_boundaries_to_type_files">
        <property name="text">
         <string>Export Plate boundary segments to files based on feature type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>(%P='plate_boundaries', %P='plate_ridge_transform_boundaries', 'plate_subduction_boundaries',
'plate_subduction_boundaries_sL', 'plate_subduction_boundaries_sR')</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_network_boundaries">
     <property name="title">
      <string>Networks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_export_individual_network_polygon_files">
        <property name="text">
         <string>Export each Network polygon to a separate file (%P='network_'+rotation id)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_network_polygons_to_single_file">
        <property name="text">
         <string>Export all Network polygons to a single file (%P='network_polygons')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_network_boundaries_to_type_files">
        <property name="text">
         <string>Export Network boundary segments to files based on feature type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>(%P='network_boundaries', %P='network_ridge_transform_boundaries', 'network_subduction_boundaries',
'network_subduction_boundaries_sL', 'network_subduction_boundaries_sR')</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_slab_polygons">
     <property name="title">
      <string>Slabs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_export_individual_slab_polygon_files">
        <property name="text">
         <string>Export each Slab polygon to a separate file (%P='slab_' + rotation id)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_slab_polygons_to_single_file">
        <property name="text">
         <string>Export all Slab polygons to a single file (%P='slab_polygons')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_slab_boundaries_to_type_files">
        <property name="text">
         <string>Export Slab boundary segments to files based on feature type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>(%P='slab_edges', %P='slab_edges_leading', 'slab_edges_leading_sL', 'slab_edges_leading_sR',
'slab_edges_trench', 'slab_edge_side')</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_all_polygons">
     <property name="title">
      <string>All Polygons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Plate/network/slab polygons combined into a single file (%P='polygons')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_plate_polygons_to_all_polygons_file">
        <property name="text">
         <string>Include all Plate polygons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_network_polygons_to_all_polygons_file">
        <property name="text">
         <string>Include all Network polygons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_slab_polygons_to_all_polygons_file">
        <property name="text">
         <string>Include all Slab polygons</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_all_boundary_segments">
     <property name="title">
      <string>All Boundary Segments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plate/network/slab boundary segments combined into files based on feature type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>(%P='boundaries', %P='ridge_transform_boundaries', 'subduction_boundaries',
'subduction_boundaries_sL', 'subduction_boundaries_sR')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_plate_boundaries_to_all_boundaries_file">
        <property name="text">
         <string>Include all Plate boundary segments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_network_boundaries_to_all_boundaries_file">
        <property name="text">
         <string>Include all Network boundary segments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_export_slab_boundaries_to_all_boundaries_file">
        <property name="text">
         <string>Include all Slab boundary segments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
